app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -1231,6 +1231,18 @@ fun scrollToPreviousParentComment(
     }
 }
 
+/**
```

Comment:
Someone should really make a library for this, if its functionality isn't already included in URL.

Reply:
But this is the java ecosystem, of course something like this exists already.

```kt
Patterns.WEB_URL.matcher(url).matches() // To check if a valid URL
println(URL("https://lemmy.ml").host) // lemmy.ml
```

Reply:
Perfect